docker-compose.yml

```diff
@@ -1,25 +1,28 @@
-version: "3.3"
+name: sonarqube_creedengo_ios
+
 services:
   sonar:
-    build:
-      dockerfile: Dockerfile
-      context: .
+    build: .
     container_name: sonar_creedengo_ios
     ports:
-      - "9000:9000"
+      - ":9000"
```

Comment:
Dynamic ports does not seem to work on macOS : I see not port binding when inspecting with Docker desktop

Reply:
please see my comments on Slack